internal/api/vm/vm.go

```diff
@@ -30,12 +30,18 @@ type ServiceOfferingMetadata struct {
 }
 
 type ServiceInstances struct {
-	Items []ServiceInstance `json:"items"`
+	NumItems int               `json:"numItems"`
```

Comment:
Why do you change the struct and introduce redundant field?

Reply:
Done for consistency with what has been done for service offerings.

Reply:
Needs further discussion since secrets do not contain NumItems and we still have inconsistency.

Reply:
Outdated. I am not changing this struct in my PR anymore.